modules/security-compliance-nist.adoc

```diff
@@ -8,22 +8,21 @@
 = Understanding compliance and risk management
 
 ifeval::["{context}" == "understanding-sandboxed-containers"]
+
+Red Hat {product-title} is designed for FIPS. When running on {op-system-base} or {op-system-first} booted in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries submitted to NIST for FIPS Validation on only the `x86_64`, `ppc64le`, and `s390X` architectures. For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status for the individual versions of the {op-system-base} cryptographic libraries submitted for validation, see link:https://access.redhat.com/articles/2918071#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards].
```

Comment:
The following is in our docs, and the language comes directly from Kirsten Newcomer. I'm not sure if you want to mimic it 100% or not, or if you're required to, but this is what is currently in OCP: 

```suggestion
{product-title} is designed for FIPS. When running {op-system-base-full} or {op-system-first} booted in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the `x86_64`, `ppc64le`, and `s390x` architectures.

For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status for the individual versions of {op-system-base} cryptographic libraries that have been submitted for validation, see link:https://access.redhat.com/articles/2918071#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards].
```